cmd/nerdctl/container_run_linux_test.go

```diff
@@ -70,6 +70,66 @@ func TestRunShmSize(t *testing.T) {
 	base.Cmd("run", "--rm", "--shm-size", shmSize, testutil.AlpineImage, "/bin/grep", "shm", "/proc/self/mounts").AssertOutContains("size=32768k")
 }
 
+func TestRunShmSizeIPCShareable(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+	const shmSize = "32m"
+
+	container := testutil.Identifier(t)
+	base.Cmd("run", "--rm", "--name", container, "--ipc", "shareable", "--shm-size", shmSize, testutil.AlpineImage, "/bin/grep", "shm", "/proc/self/mounts").AssertOutContains("size=32768k")
+	defer base.Cmd("rm", "-f", container)
+}
+
+func TestRunIPCShareableRemoveMount(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+	container := testutil.Identifier(t)
+
+	base.Cmd("run", "--name", container, "--ipc", "shareable", testutil.AlpineImage, "sleep", "0").AssertOK()
+	result := base.Cmd("rm", container).Run()
+	fmt.Println(result.Stdout())
+	fmt.Println(result.Stderr()) // time="2023-12-03T12:53:17Z" level=warning msg="failed to remove container state dir ~/.local/share/nerdctl/1935db59/containers/nerdctl-test/cc44924f9b1af6cd22f599c1bb11303b82b7dd076ce09777b048bda84e744b7e" error="<nil>"
```

Comment:
The `fmt.Println` may be used for debug.
As a test, The `fmt.Println` may be replaced by `AssertOK` is better :-)

Reply:
Applied. Thanks!